src/__mocks__/mocks.ts

```diff
@@ -47,3 +47,8 @@ export function getTermAndServices(lang: string): string {
     return getTermAndServices('it');
   }
 }
+
+export function track(event_name: string, properties?: any): void {
```

Comment:
@pasqualespica Wouldn't it be nice if we add a unit/integration test for each page which truly mocks `mixpanel.track(...)` using `jest.mock(...)`?